MBOM/en/0x30-Use_Case_Software_Simple.md

```diff
@@ -69,63 +69,59 @@ The `component` objects are defined as follows:
 
 #### Source components
 
-- `helloworld.c`:
-    </br>
```

Comment:
not needed, markdown knows how to do line-breaks: add two white spaces at the line ends.so this was used instead.